examples/optical-character-recognition/run_trocr.py

```diff
@@ -0,0 +1,243 @@
+import argparse
+import os
+import time
+
+import pandas as pd
+import torch
+from datasets import load_metric
+from PIL import Image
+from torch.utils.data import DataLoader, Dataset
+from transformers import TrOCRProcessor, VisionEncoderDecoderModel
+
+from optimum.intel.neural_compressor import IncOptimizer, IncQuantizationConfig, IncQuantizationMode, IncQuantizer
+from optimum.intel.neural_compressor.quantization import IncQuantizedModelForVision2Seq
+
+
+parser = argparse.ArgumentParser(description="PyTorch ImageNet Training")
```

Comment:
I think it would be nice to use `HfArgumentParser` and create classes such as `ModelArguments` and `OptimizationArguments` to stay consistent with the other examples scripts

Reply:
done